render_test.go

```diff
@@ -34,18 +35,57 @@ func TestRenderPageMethod(t *testing.T) {
 			renderer.BuildPage(w, mockPage, "")
 
 			Convey("Then the render client should set an error", func() {
-				So(mockClient.ValidBuildHTMLMethodCalls, ShouldEqual, 0)
+				So(len(mockClient.ValidBuildHTMLMethodCalls), ShouldEqual, 0)
```

Comment:
Could use `ShouldBeEmpty`
```suggestion
				So(mockClient.ValidBuildHTMLMethodCalls, ShouldBeEmpty)
```

Reply:
Think this was an exisitng test but updated 👍